### ISSUE TITLE
The pretrained checkpoint of grasp task

### ISSUE THREAD

**Issue by zhiyuanYU134** [hgaurav2k/hop#5]
Hi, what a great job! I used the provided checkpoint to train the grasp task. But it does not perform well. So could you nicely provide a pretrained checkpoint of the grasp task for testing? My email is yuzhiyuan317@gmail.com.

**Comment by zhiyuanYU134** [comment]
These are some curves about the training of grasp task.
![2](https://github.com/user-attachments/assets/f5d01e8b-141f-43af-aad1-ddc1955bdc1d)
![捕获](https://github.com/user-attachments/assets/f6136796-be25-47b3-9cd4-26c3d77cb2d4)
![捕获1](https://github.com/user-attachments/assets/82faaf70-c558-4487-a9b8-47c8b5303aa8)

**Comment by hgaurav2k** [comment]
Hi, 

Thanks for checking out our work! 

Comparing with the plots in our paper, I see that you get the same success curve as ours!
<img width="395" alt="Screenshot 2024-10-16 at 12 55 12 PM" src="https://github.com/user-attachments/assets/4f74c15b-3fac-4cc0-a590-83da70b2d547">

Could you please explain which part is not working well for you?

**Comment by zhiyuanYU134** [comment]
Thanks for your reply. When i test the model of grasp task, the robot runs slowly and does not grasp the object.

https://github.com/user-attachments/assets/9e6a079c-033d-44da-91b0-ef93844c7601

The command that i used as follows:
finetune_grasp.sh:
cmd="python scripts/finetune.py   num_gpus=1 \
    checkpoint="outputs/pretrain/prior.pt"\
    task=AllegroXarmNew \
    train.algo=PPOTransformer \
    train.ppo.initEpsHand=0.1 \
    train.ppo.initEpsArm=0.1 \
    train.ppo.learning_rate=1e-5 \
    train.ppo.value_grads_to_pointnet=False \
    train.ppo.critic_warmup_steps=200 \
    wandb_activate=True  wandb_name=AllegroXarmGrasping_finetune_datamix_pretraining\
    pipeline=gpu  rl_device=cuda:0  sim_device=cuda:0 \
    train.ppo.minibatch_size=512 num_envs=512 \
    seed=-1"
run_policy.sh:
POLICY="outputs/AllegroXarmGrasping_finetune_datamix_pretraining/2024-10-11_23-33/stage1_nn/last.pth" #"outputs/AllegroXarmGrasping_scratch_vel_control/2024-05-29_00-49/stage1_nn/ep_41700_step_1708M_reward_1876.28.pth"
cmd="python scripts/finetune.py   num_gpus=1 \
    task=AllegroXarmNew test=True headless=False \
    task.env.useOldActionSpace=True \
    checkpoint=$POLICY  \
    train.algo=PPOTransformer \
    wandb_activate=False  wandb_name=AllegroXarmGrasping_Scratch \
    pipeline=gpu  rl_device=cuda:0  sim_device=cuda:0 \
    train.ppo.minibatch_size=16 num_envs=16 \
    task.env.episodeLength=1000 \
    task.env.maxConsecutiveSuccesses=1 \
    pc_input=True \
    seed=-1"
These are some figures about this training:
![reward_analysis_AllegroXarmGrasping_Scratch](https://github.com/user-attachments/assets/eb03a9dd-ecf3-46d9-879d-27b8356c6ada)
![success_analysis_AllegroXarmGrasping_Scratch](https://github.com/user-attachments/assets/58ed0856-0ae1-457b-beef-53e4879a2542)

**Comment by hgaurav2k** [comment]
I see that you have not changed the action space when going from velocity control (the PPO baseline) to the policy finetuned from the base policy (which is joint position control). As mentioned in our paper, PPO with joint position control fails to explore, so we change the action space to velocity control to get a strong baseline. Could you please try with setting the argument task.env.useOldActionSpace=False?

**Comment by hgaurav2k** [comment]
Also, thanks for pointing this out. I will explicitly add this in the script.

**Comment by zhiyuanYU134** [comment]
Thanks for your help! After setting the argument task.env.useOldActionSpace=False, the result seems very good.
![2024-10-21 01-48-32屏幕截图](https://github.com/user-attachments/assets/5f3e49a4-0b9d-4fb9-8749-1d82611cd4d7)
